sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubConsumerClient.cs

```diff
@@ -792,12 +775,25 @@ async Task performCleanup()
                     throw new EventHubsException(false, EventHubName, String.Format(CultureInfo.CurrentCulture, Resources.FailedToCreateReader, EventHubName, partitionId, ConsumerGroup));
                 }
 
+                void exceptionCallback(Exception ex)
```

Comment:
Orthogonal;  missed this case during implementation.  These are known cases that may be triggered due to non-determinism when cancelling background publishing for multiple partitions.  For a single iterator, it would indicate an exception but for this case, it is a result of the benign race.